tests/unit_tests/udfs/test_sequence_classification_single_text.py

```diff
@@ -53,37 +53,17 @@ def create_mock_metadata(udf_wrapper):
 
 
 @pytest.mark.parametrize("params", [
-    SingleModelSingleBatchComplete(),
-    SingleModelSingleBatchIncomplete(),
-    SingleModelMultipleBatchComplete(),
-    SingleModelMultipleBatchIncomplete()
-])
-def test_sequence_classification_single_text_single_model(
-        params, upload_dummy_model_to_local_bucketfs):
-    model_metadata = upload_dummy_model_to_local_bucketfs[0]
-
-    executor = UDFMockExecutor()
-    meta = create_mock_metadata(params.udf_wrapper)
-    bucketfs_connection = Connection(address=f"file://{model_metadata[1]}")
-    exa = MockExaEnvironment(
-        metadata=meta,
-        connections={BFS_CONN_NAME: bucketfs_connection})
-
-    input_data = []
-    for text in params.single_text:
-        input_data.append((
-            BFS_CONN_NAME, model_metadata[0], model_metadata[1], text))
-    result = executor.run([Group(input_data)], exa)
-
-    rounded_result = _get_rounded_result(result)
-    expected_result = _prepare_expected_result(input_data)
-    assert rounded_result == expected_result
-
-
-@pytest.mark.parametrize("params", [
-    MultipleModelMultipleBatchComplete
+    SingleModelSingleBatchComplete,
+    SingleModelSingleBatchIncomplete,
+    SingleModelMultipleBatchComplete,
+    SingleModelMultipleBatchIncomplete,
+    MultipleModelMultipleBatchComplete,
+    MultipleModelMultipleBatchIncomplete,
+    MultipleModelSingleBatchComplete,
+    MultipleModelSingleBatchIncomplete,
+    MultipleModelMultipleBatchMultipleModelsPerBatch
 ])
-def test_sequence_classification_single_text_multiple_models_with_factory(
+def test_sequence_classification_single_text(
         params, upload_dummy_model_to_local_bucketfs):
     models_metadata = upload_dummy_model_to_local_bucketfs
```

Comment:
upload_dummy_model_to_local_bucketfs created files which we don't need for this test, please have a look at it, if we can simplify this

Reply:
we need a bucketfs_location in UDF. That's why we have to create a valid localfs in tests. But I updated it by removing redundant part in the fixture